auslib/blobs/apprelease.py

```diff
@@ -115,8 +118,14 @@ def _getSpecificPatchXML(self, patchKey, patchType, patch, updateQuery, whitelis
         if isForbiddenUrl(url, updateQuery['product'], whitelistedDomains):
             return None
 
-        return '        <patch type="%s" URL="%s" hashFunction="%s" hashValue="%s" size="%s"/>' % \
+        patchXML = '        <patch type="%s" URL="%s" hashFunction="%s" hashValue="%s" size="%s"' % \
             (patchType, url, self["hashFunction"], patch["hashValue"], patch["filesize"])
+        additionalPatchAttributes = self._getAdditionalPatchAttributes(patch)
+        for attribute in additionalPatchAttributes:
+            patchXML += (' ' + attribute + '="' + additionalPatchAttributes[attribute] + '"')
```

Comment:
This gets the right result, but please follow the existing style of string construction in this file (using %s for variables like in line 121).